contents/tests.py

```diff
@@ -1,3 +1,72 @@
 from django.test import TestCase
 
-# Create your tests here.
+from contents.models import Contents
+
+
+class ContentsCreateAPITest(TestCase):
```

Comment:
HTTP 415 에러 아래 처럼 수정하면 고쳐짐

```suggestion
class ContentsCreateAPITest(TestCase):
    client_class: APIClient = APIClient
```